lookout/style/format/rules.py

```diff
@@ -158,7 +159,8 @@ def __init__(self,
                  base_model: Union[DecisionTreeClassifier, RandomForestClassifier],
                  prune_branches: bool = True, prune_branches_algorithm: str = "top-down-greedy",
                  top_down_greedy_budget: TopDownGreedyBudget = (False, 1.0),
-                 prune_attributes: bool = True, uncertain_attributes: bool = True):
+                 prune_attributes: bool = True, uncertain_attributes: bool = True,
+                 prune_base_model: bool = False):
```

Comment:
remove this please, we already have `prune_branches_algorithm`. We will never combine these two.